web/src/components/Storyblok/StoryblokAuthor/AuthorList/AuthorList.test.tsx

```diff
@@ -0,0 +1,183 @@
+import { StoryblokComponent } from "@storyblok/react";
+import { render, screen } from "@testing-library/react";
+import { StoryblokStory } from "storyblok-generate-ts";
+
+import { AuthorStoryblok } from "@/types/storyblok";
+
+import { StoryblokAuthor } from "../StoryblokAuthor";
+
+import { AuthorList, AuthorListProps } from "./AuthorList";
+
+jest.mock("@storyblok/react", () => ({
+	StoryblokComponent: ({ blok }: { blok: AuthorStoryblok }) => {
+		//
+		return <StoryblokAuthor blok={blok} />;
+	},
+}));
+
+const authorsListContent: AuthorListProps = {
+	authors: [
```

Comment:
When/if you can be bothered, you might want to swap this out so it makes use of the test utility objects I've just added. Totally optional though, as long as the test is doing its job